### DISCUSSION TITLE
mestiez should go through some old suggestions

### DISCUSSION THREAD

**Discussion by TeslaCoil9001** [studio-minus/ppg-bugs-and-requests#2053] [Ideas]
It seems like some old suggestions got lost to time, which is a shame since some are really useful or fun sounding. Here are some examples:

https://github.com/studio-minus/ppg-bugs-and-requests/issues/315
https://github.com/studio-minus/ppg-bugs-and-requests/issues/153
https://github.com/studio-minus/ppg-bugs-and-requests/issues/134
https://github.com/studio-minus/ppg-bugs-and-requests/issues/128
https://github.com/studio-minus/ppg-bugs-and-requests/issues/312
https://github.com/studio-minus/ppg-bugs-and-requests/issues/313

Maybe Mestiez should consider looking through old suggestions and accepting or closing some. There are old suggestions with features that were added later on and are still open.

It's just a thought though, I'm not pressuring you to do it, but I feel as if it would be a good idea.

**Comment by dinevolence** [comment]
Stop bumping literally everything, it's just comment spam. zooi already has some of them planned already, they're just still pending

**Reply by TeslaCoil9001** [reply]
soz
